aries_cloudagent/config/argparse.py

```diff
@@ -2053,11 +2053,20 @@ def add_arguments(self, parser: ArgumentParser):
             ),
         )
 
+        parser.add_argument(
+            "--force-upgrade",
+            action="store_true",
+            env_var="ACAPY_UPGRADE_FORCE_UPGRADE",
+            help="Brute force the upgrade process.",
```

Comment:
Should be — “Forces the '—from-version' argument to override the version retrieved from secure storage when calculating upgrades to be run."